environments/base/manifests/thanos-query-frontend-serviceMonitor.yaml

```diff
@@ -1,26 +0,0 @@
-apiVersion: monitoring.coreos.com/v1
```

Comment:
We probably want to keep this file and forgot an `serviceMonitor: true` somewhere? :thinking:

Reply:
On the contrary, we don't generate them in here. This is the only instance. I removed it to be consistent.